src/ebay-button/button.tsx

```diff
@@ -71,17 +67,24 @@ const EbayButton:FC<Props> = ({
         regular: '',
         default: ''
     }
+    const splitStyles: { [key in Split]: string } = {
+        start: `${classPrefix}--split-start`,
+        end: `${classPrefix}--split-end`
+    }
     const isDestructive = variant === 'destructive'
     const isForm = variant === 'form'
     const isLoading = bodyState === 'loading'
+    const isExpand = bodyState === 'expand'
+    const isSlim = isForm && (isIconOnly(children) || (isExpand && !children))
```

Comment:
@darkwebdev looks like you removed this method here

https://github.com/eBay/ebayui-core-react/pull/166/files#diff-4d320ba8706cd7971e755d3ceae561244c47b6e4e95514fe7ce84b22eb155b14L37-L41

Reply:
nvm - noticed that its at the end :)

https://github.com/eBay/ebayui-core-react/pull/166/files#diff-4d320ba8706cd7971e755d3ceae561244c47b6e4e95514fe7ce84b22eb155b14R146-R149